### PR TITLE
CedricLouvrier.Pinga version 0.64.1

### PR DESCRIPTION
**Pull request by ye4241**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest .\manifests\c\CedricLouvrier\Pinga\0.64.1\`?
- [x] Have you tested your manifest locally with `winget install --manifest .\manifests\c\CedricLouvrier\Pinga\0.64.1\`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/170540)

### CODE REVIEW

**Comment by ye4241** [comment]
Same issue as #170538

**Comment by stephengillie** [comment]
> Same issue as #170538

https://github.com/microsoft/winget-pkgs/pull/170538#issuecomment-2318443659

This would probably impact all packages from this developer, and possibly all packages on this host.

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue
[Policy] Needs-Author-Feedback

File: manifests/c/CedricLouvrier/Pinga/0.64.1/CedricLouvrier.Pinga.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: CedricLouvrier.Pinga
+PackageVersion: 0.64.1
+InstallerType: zip
+NestedInstallerType: inno
+NestedInstallerFiles:
+- RelativeFilePath: pinga.exe
+Installers:
+- Architecture: x64
+  InstallerUrl: https://css-ig.ye4241.workers.dev/bin/pinga-win64.zip
```
Comment (ItzLevvie) [inline manifests/c/CedricLouvrier/Pinga/0.64.1/CedricLouvrier.Pinga.installer.yaml:12]:
```suggestion
  InstallerUrl: https://css-ig.net/bin/pinga-win64.zip
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/c/CedricLouvrier/Pinga/0.64.1/CedricLouvrier.Pinga.installer.yaml (hunk 1 above)
Comment (ye4241) [reply manifests/c/CedricLouvrier/Pinga/0.64.1/CedricLouvrier.Pinga.installer.yaml:12]: https://github.com/microsoft/winget-pkgs/pull/170538#discussion_r1748092198

Same issue here.

**Comment by denelon** [comment]
This PR is using an older version of the schema. I'm going to go ahead and close this for now. Once the manifest has been updated, either create a new PR or re-open this one.

**Pull request closed without merging**